modules/arm_plugin/src/arm_converter/arm_converter_gather.cpp

```diff
@@ -8,6 +8,25 @@
 #include "arm_converter/arm_converter.hpp"
 
 namespace ArmPlugin {
+template <> Converter::Conversion::Ptr Converter::Convert(const opset::Gather& node) {
```

Comment:
I think we can create an transformation opset::Gather to opset::ArmGather for some cases to infer it via ACL

Reply:
As far as I could understand there already is [one](https://github.com/openvinotoolkit/openvino_contrib/blob/3e516d960d11f69a67dd30e87a5f37c2a3e7e94d/modules/arm_plugin/src/transformations/convert_gather_arm.cpp#L15)